Packages/vcs/Lib/vtk_ui/text.py

```diff
@@ -49,34 +158,24 @@ def text_actor(string, fgcolor, size, font):
 
     return actor
 
-def text_dimensions(text, text_prop):
-    ren = vtkTextRenderer()
-    bounds = [0,0,0,0]
-    ren.GetBoundingBox(text_prop, text, bounds)
-    return bounds[1] - bounds[0], bounds[3] - bounds[2]
 
-def baseline_offsets(origin, new_string, text_prop):
+def text_dimensions(text, text_prop, at_angle=0):
     ren = vtkTextRenderer()
+    bounds = [0, 0, 0, 0]
+    p = vtkTextProperty()
+    p.ShallowCopy(text_prop)
+    p.SetOrientation(at_angle)
+    ren.GetBoundingBox(p, text, bounds)
+    return bounds[1] - bounds[0], bounds[3] - bounds[2]
```

Comment:
The dimensions would actually be `bounds[1] - bounds[0] + 1, bounds[3] - bounds[2] + 1`, as the vtkTextRenderer method returns the pixel extent of the rendered image. (note that the vtkTextActor's bounding box accounts for the '+1" already)